app/helpers/users_helper.rb

```diff
@@ -15,19 +15,19 @@ def options_for_roles
   end
 
   def options_for_partners
-    policy_scope(Partner).all.order(:name).pluck(:name, :id)
```

Comment:
Removed scoping so users can see partners they are assigned to. Should revert this since removing the disabled elements code.